src/index.tsx

```diff
@@ -15,35 +16,17 @@ import './css/common/pp-semantic-ui-reset.scss';
 import './css/common/pp-semantic-ui-overrides.scss';
 
 import './css/selection.scss';
-import {showEditorNewAnnotation} from 'store/widgets/actions';
-import {getAnnotationUrl} from './utils/url'
 
 console.log('Przypis script working!');
 
-function injectApp() {
-  const documentContainer = document.createElement('div');
-  documentContainer.id = 'pp-document-container';
-  window.document.body.appendChild(documentContainer);
-
-  // TODO: 1. wrap those dispatches into dedicated function to make this code more separate and descriptive
-  // TODO: 2. this url is hardcoded now, but should be imported from config
-  store.dispatch(setAxiosConfig({
-    baseURL: 'http://localhost:8000/api',
-  }));
-
-  // This our root request that need to have part of the url (path) hardcoded
+function loadData() {
+  // This is our root request that needs to have part of the url (path) hardcoded
   store.dispatch(readEndpoint('/annotations?url=' + getAnnotationUrl()));
-
-  ReactDOM.render(
-    <Provider store={store}>
-      <App />
-    </Provider>,
-    documentContainer,
-  );
 }
 
 const isBrowser = typeof window !== 'undefined';
 if (isBrowser) {
   initializeCoreHandlers();
-  injectApp();
+  injectComponents();
+  loadData();
```

Comment:
How about referring to _init_? It would be nice to have it in that name as it is that one particular data loading. My first try is `loadInitialData`, but that's just a try...

Reply:
:+1: